controllers/api/quota.py

```diff
@@ -184,9 +184,9 @@ def GET(self):
             'groupusedmem'      : groupusedmem,
             'groupusedsys'      : groupusedsys,
             'availablevm'   : availablevm,
-            'availablecpu'   : availablecpu,
-            'availablemem'   : availablemem,
-            'availablesys'   : availablesys,
+            # 'availablecpu'   : availablecpu,
+            # 'availablemem'   : availablemem,
+            # 'availablesys'   : availablesys,
```

Comment:
Check it now 

 

Thanks

 

Alexander

 

From: Callum Williams [mailto:notifications@github.com] 
Sent: 01 December 2015 16:23
To: stfc/cloud <cloud@noreply.github.com>
Cc: Dibbo, Alexander (STFC,RAL,SC) <alexander.dibbo@stfc.ac.uk>
Subject: Re: [cloud] fixes syntax and casting errors (#45)

 

In controllers/api/quota.py <https://github.com/stfc/cloud/pull/45#discussion_r46299729> :

> @@ -184,9 +184,9 @@ def GET(self):
>              'groupusedmem'      : groupusedmem,
>              'groupusedsys'      : groupusedsys,
>              'availablevm'   : availablevm,
> -            'availablecpu'   : availablecpu,
> -            'availablemem'   : availablemem,
> -            'availablesys'   : availablesys,
> +            # 'availablecpu'   : availablecpu,
> +            # 'availablemem'   : availablemem,
> +            # 'availablesys'   : availablesys,

Same for this?

—
Reply to this email directly or view it on GitHub <https://github.com/stfc/cloud/pull/45/files#r46299729> .